docs/contract_types.rst

```diff
@@ -335,14 +332,24 @@ Options used by multiple contracts
 
 - ``ignore_imports``: Optional list of imports, each in the form ``mypackage.foo.importer -> mypackage.bar.imported``.
   These imports will be ignored: if the import would cause a contract to be broken, adding it to the list will cause the
-  contract be kept instead.
+  contract be kept instead. Supports wildcards (see below).
```

Comment:
Nice. I wonder if it would be worth a hyperlink to that section (a bit like we do with linking to the _Options used by multiple contracts_ section).